model/user.go

```diff
@@ -150,3 +152,59 @@ func checkPwd(password string) error {
 func checkEmail(email string) error {
 	return nil
 }
+
+type UserFilter struct {
+	ID    []string `json:"id,omitempty"`
+	Email []string `json:"email,omitempty"`
+
+	CreatedAfter  *time.Time `json:"created_after,omitempty"`
+	CreatedBefore *time.Time `json:"created_before,omitempty"`
+
+	UpdatedAfter  *time.Time `json:"updated_after,omitempty"`
+	UpdatedBefore *time.Time `json:"updated_before,omitempty"`
+}
+
+func (fltr *UserFilter) ParseForm(form url.Values) error {
```

Comment:
It would be straightforward to implement user test.go and cover these methods...

Reply:
Right! I totally forgot about these methods!